bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -616,6 +649,9 @@ int handleNavigationStarting(long pView, long pArgs, boolean top) {
 	int hr = args.get_Uri(ppszUrl);
 	if (hr != COM.S_OK) return hr;
 	String url = wstrToString(ppszUrl[0], true);
+	if(isLocationForCustomText(url)) {
+		url = ABOUT_BLANK;
+	}
```

Comment:
This logic to replace the actual with the fake URL if necessary is replicated three times. Would be good to wrap it into some internal, self-explaining `getExposedUrl()` method or the like.

Reply:
Done